stl/inc/ostream

```diff
@@ -109,7 +109,11 @@ public:
         _STL_DISABLE_DEPRECATED_WARNING
         __CLR_OR_THIS_CALL ~sentry() noexcept {
 #if _HAS_EXCEPTIONS
+#if _HAS_DEPRECATED_UNCAUGHT_EXCEPTION

```

Comment:
This is still blocked by ArchivedOS-12000909. @stwish-msft Do you know what the status of that is?

Reply:
Any update here?